services/ui-src/src/components/Footer/Footer.jsx

```diff
@@ -1,67 +1,79 @@
+                    />
+                  </div>
+                </div>
+                <div className="footerCMSBrandingRight">
+                  <p className="hhsCopyText">
+                    A federal government website managed and paid for by the
+                    U.S. Centers for Medicare and Medicaid Services and part of
+                    the MDCT suite.
+                  </p>
+                </div>
+              </div>
+              <div className="footerCMSMedicaid">
+                <div className="medicaidLogo">
                   <img
-                    src="/img/seds-logo.svg"
-                    alt="MDCT SEDS: Statistical Enrollment Data Systems, Medicaid Data Collection Tool"
+                      src="/img/logo_medicaid.svg"
+                      alt="Medicaid.gov: Keeping America Healthy"
                   />
-                </li>
-              </ul>
-            </div>
-
-            <div className="tagline">
-              Centers for Medicare &amp; Medicaid Services
-            </div>
+                </div>
+              </div>
+            </Grid>
           </Grid>
-          <Grid col={6} tablet={{ col: true }}>
-            <div className="footer-nav">
-              <FooterNav
-                aria-label="Footer navigation"
-                size="slim"
-                links={[
-                  <a
-                    className="usa-footer__primary-link"
-                    href="https://www.cms.gov/"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    Centers for Medicare &amp; Medicaid Services Website
-                  </a>
-                ]}
-              />
+        </GridContainer>
+      </div>
+      <div className="footerBottom">
+        <div className="footerBottomContainer">
+          <div className="footerBottomFlex">
+            <div className="footerBottomLinkFlex">
+              <a href="mailto:mdct_help@cms.hhs.gov" className="link">
+                Contact Us
+              </a>
+              <a
+                href="https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/CMSNondiscriminationNotice"
+                target="_blank"
+                className="link"
+              >
+                Accessibility Statement
+              </a>
             </div>
-            <div className="info">
-              <div className="help">
-                <p>
-                  Email{" "}
-                  <a
-                    data-test="attribute-email"
-                    href="mailto:mdct_help@cms.hhs.gov"
-                  >
-                    MDCT_Help@cms.hhs.gov
-                  </a>{" "}
-                  for help or feedback.
-                </p>
-              </div>
-              <div className="title">
-                A federal government managed website by the Centers for Medicare
-                &amp; Medicaid Services
-              </div>
+            <div>
```

Comment:
This `<div>` needs to have `display: flex;` on it so that it centers the address text. Right now it's slightly misaligned.

https://github.com/user-attachments/assets/491f95c6-ed95-4ad1-9e70-2af0a4e674ca